codefresh/resource_pipeline.go

```diff
@@ -106,6 +106,10 @@ func resourcePipeline() *schema.Resource {
 										Optional: true,
 										Default:  "github",
 									},
+									"description": {
```

Comment:
Event if we supported this field for the pipeline, why do you put in `spec_template`?